src/content/competitions/HiPAC2023.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Championship and second place at HiPAC 2023 competition"
+description: "NTHU Team-2 and Team-ZY secure the champion and second place respectfully at 2nd HIGH PERFORMANCE APPLICATION COMPETITION 2023"
+pubDate: "Aug 10 2023"
+heroImage: "/HiPAC-2023-banner.jpg"
+---
+Our student team from National TsingHua University's Computer Science department have secured the champion of 2nd HIGH PERFORMANCE APPLICATION COMPETITION again, with a second place also taken done by us!
```

Comment:
"National TsingHua University" should be replaced with "National Tsing Hua University"
The name of Tsinghua University in Beijing should be written together as one word.

Reply:
fixed it!